### PR TITLE
mupdf refactoring cfr. https://github.com/netblue30/firejail/discussions/4993

### PR DESCRIPTION
**Pull request by glitsj16**

### CODE REVIEW

File: etc/profile-m-z/mupdf-x11-curl.profile [hunk 1]
```diff
@@ -8,9 +8,9 @@ include mupdf-x11-curl.local
 #include globals.local
 
 ignore net none
-
 netfilter
 protocol unix,inet,inet6
+ignore protocol
```
Comment (rusty-snake) [inline etc/profile-m-z/mupdf-x11-curl.profile:13]: Not needed, `protocol` accumulates.

File: etc/profile-m-z/mupdf-x11-curl.profile [hunk 2]
```diff
@@ -8,9 +8,9 @@ include mupdf-x11-curl.local
 #include globals.local
 
 ignore net none
-
```
Comment (rusty-snake) [inline etc/profile-m-z/mupdf-x11-curl.profile:11]: I think this is intentionally to keep `ignore`s in it's own block.

File: etc/profile-m-z/mupdf.profile [hunk 3]
```diff
@@ -41,3 +42,6 @@ private-tmp
 
 dbus-user none
 dbus-system none
+
+memory-deny-write-execute
```
Comment (rusty-snake) [inline etc/profile-m-z/mupdf.profile:46]: Sure that this does not break `mupdf-gl`?

File: etc/profile-m-z/mupdf-x11-curl.profile (hunk 1 above)
Comment (glitsj16) [reply etc/profile-m-z/mupdf-x11-curl.profile]: Fixed, thanks.

File: etc/profile-m-z/mupdf.profile (hunk 3 above)
Comment (glitsj16) [reply etc/profile-m-z/mupdf.profile:46]: I've checked several times and both mupdf and mupdf-gl both work with mdwe. At least they do on Arch Linux...

**Review by rusty-snake — APPROVED** [review]

**Pull request merged**